crdt/causal_tree/functions.go

```diff
@@ -0,0 +1,19 @@
+package causal_tree
+
+import (
+	atm "github.com/crdteam/causal-tree/crdt/atom"
+	"github.com/google/uuid"
+)
+
+// NewCausalTree creates an initialized empty replicated tree.
+func NewCausalTree() *CausalTree {
```

Comment:
Use just `New`, so that the usage is `causaltree.New`

Reply:
done.